src/core/main.cpp

```diff
@@ -223,7 +224,17 @@ void fileCheck( QString &file )
 	}
 }
 
+int usageError(const QString& message)
+{
+	qCritical().noquote() << QString("\n%1.\n\nTry \"%2 --help\" for more information.\n\n")
+				   .arg( message ).arg( qApp->arguments()[0] );
+	return EXIT_FAILURE;
+}
 
+int noInputFileError()
+{
+	return usageError( "No input file specified" );
```

Comment:
This one doesn't finish with '.', while the others do. As @Wallacoloo said, I think you should add a period here and remove one in `usageError`.